src/components/pages/playlists/PlaylistPlayer.vue

```diff
@@ -316,27 +317,25 @@
       ref="video-progress"
       class="video-progress pull-bottom"
       :annotations="annotations"
+      :entity-ist="entityList"
+      :empty="!isCurrentPreviewMovie"
       :fps="fps"
       :frame-duration="frameDuration"
-      :is-playlist="true"
       :is-full-mode="isFullMode"
       :is-full-screen="fullScreen || isEntitiesHidden"
       :movie-dimensions="movieDimensions"
-      :nb-frames="nbFrames"
+      :nb-frames="
+        isCurrentPreviewMovie ? nbFrames : isCurrentPreviewPicture ? 48 : 0
```

Comment:
Why the number "48"?
maybe add a named constant to be clearer.

Reply:
It's arbitrary. It raises many questions. I prefer to keep it clumsy once we address the question of picture duration more properly.